airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -39,6 +39,8 @@ acceptance_tests:
             bypass_reason: Unable to populate
           - name: quotes
             bypass_reason: Unable to populate
+          - name: contacts_merged_audit
+            bypass_reason: Unable to populate
```

Comment:
I'm uncertain whether it's possible to merge any stream without conducting tests. How can we determine if it's functioning properly?

@marcosmarxm wdyt?